### DISCUSSION TITLE
New year, new quarter, new month

### DISCUSSION THREAD

**Discussion by tamarw** [iSoron/uhabits#1610] [Help & FAQ]
I have a habit for reading that should technically restart today for the week, month, quarter, and year, right? 

But in the attached, the month, quarter, and year are showing random numbers from other data sources. Even the week is off by 3 and I didn't even read 3 pages yesterday.

Not sure where this is coming from.

![Screenshot_20230101-200321](https://user-images.githubusercontent.com/14209647/210189177-24b0938f-ce54-4a53-a9cb-828be1ed883e.png)

**Comment by iSoron** [comment]
The chart is indicating that you read 16 pages today, this week, this quarter and this year. For this week, there are still 19 pages to go; for the month, there are 139 pages to go, and so on.
